assets/src/stories-editor/blocks/amp-story-page/edit.js

```diff
@@ -275,6 +292,18 @@ class PageEdit extends Component {
 					</PanelColorSettings>
 					<PanelBody title={ __( 'Background Media', 'amp' ) }>
 						<>
+							{
+								isExcessiveVideoSize &&
+								<Notice status="warning" isDismissible={ false } >
+									{
+										sprintf(
+											/* translators: %d: the number of bytes in a megabyte */
+											__( 'A video size of less than %d MB per second is recommended.', 'amp' ),
```

Comment:
This seems to be working:

![image](https://user-images.githubusercontent.com/134745/59155961-782cef80-8a48-11e9-8253-c7e6192cc004.png)

Reply:
Nice!

Reply:
Just the translator comment is incorrect (bytes vs megabytes)

Reply:
Fixed in bb9e2d0de4e90707fde00621ed4f4338e8e6ddf0